### ISSUE TITLE
Focus loss on username/password in expo web

### ISSUE THREAD

**Issue by hew** [aws-amplify/amplify-js#4483]
**Describe the bug**
Focus loss on username/password fields using expo web. 

**To Reproduce**

1. initialize fresh expo project 
2. `npm i aws-amplify-react-native`
3. Try to use the SignIn component:
```
<Authenticator
  amplifyConfig={myAWSExports}
>
  <SignIn>
</Authenticator>
```
4. `expo start --web`

At this point, you should be able to attempt some input in the browser. Immediately, the fields will lose focus, preventing you from entering any text.

**Expected behavior**
The expected behaviour is that the input fields should NOT lose focus.

**Comment by hew** [comment]
Note: this only happens with the `<SignIn />` component. Not any of the others.

**Comment by davidbiller** [comment]
For web, use the aws-amplify-react component. 
you can add a postfix to the js files like:
login.js
login.web.js

**Comment by ericclemmons** [comment]
This sounds similar to #3827.  @hew Did @davidbiller's suggestion work, with using `aws-amplify-react` via `login.web.js`? 🤞

In the meantime, I'll need to setup a project and reproduce it...

**Comment by ericclemmons** [comment]
I was able to reproduce it here:
> https://github.com/ericclemmons/amplify-js-issues-4483

Using a vanilla `<TextInput />` works fine, so there's something going on here that's causing the focus to be lost.

Going to mark this as a bug & get some 👀 on this...

**Comment by ericclemmons** [comment]
Reproduced with the `react-native` CLI as well:
> ![react-native focus](https://user-images.githubusercontent.com/15182/71040090-a287f580-20da-11ea-825a-f1b12ff994c0.gif)

**Comment by ericclemmons** [comment]
Aha! So the problem is with using `<Authenticator>` instead of `withAuthenticator(App)`.

There isn't React Native support for `<Authenticator>` (or its sub-components), only the Higher-order Component (HoC):
> https://aws-amplify.github.io/docs/js/react#add-auth

(Or, based on your usage, `Authenticator` and `SignIn` are broken)

@hew Try converting your code to use `withAuthenticator(App)` and let us know if that resolves it for you.

I'm looping in a few others on the team for extra 👀 on this.

**Comment by hew** [comment]
@ericclemmons Thanks for the response! 😀 

I ended up going with a custom approach using the `SignIn` method directly. So I don't have any immediate need, here. Thanks again.

**Comment by ericclemmons** [comment]
@hew Glad to hear you're unblocked 😅 Going to keep this open so that we can at least get this addressed in the future with #3279.

**Comment by GeorgeBellTMH** [comment]
So switching the View to a ScrollView in SignIn.js resolves the issue...

<Header theme={theme}>{I18n.get('Sign in to your account')}</Header>
					<View style={theme.sectionBody}>

<Header theme={theme}>{I18n.get('Sign in to your account')}</Header>
					<ScrollView style={theme.sectionBody}>

**Comment by amhinson** [comment]
Just merged a fix for this, and it should be in the next release. I'm going to go ahead and close this issue for now, but feel free to open a new issue if the issue persists 👍
